src/Sync.php

```diff
@@ -129,8 +130,11 @@ public function import()
     foreach ($files as $type => $file) {
       $json = json_decode(file_get_contents($file), true);
 
-      if (!empty($json) && isset($json) && $json['data']) {
-        $payload[$type] = $json['data'];
+      if (!empty($json) && isset($json) &&
+        isset($json['data']) && $json['data'] &&
+        isset($json['type']) && $json['type']) {
```

Comment:
Is there a need to check for `$json['type']`'s existence here? It gets passed in via $type which is defined back up in line 95. Happy to replace the current check of `!$json['type']` with `isset($json['type']) && $json['type'])` on line 90 though since that probably makes more sense